types/slideshow.go

```diff
@@ -0,0 +1,13 @@
+package types
+
+type SlideShowResponse struct {
+	Speed      int
+	FileImages []string
+	UniqueTags []string
+}
+
+type SlideShowConfig struct {
+	Speed        int
+	IncludedTags []string
+	UniqueTags   []string
```

Comment:
What's the difference between these 2 fields?

Reply:
IncludedTags are the tags that the user selected to be shown in the current slide show, uniquetags is the list of all possible tags they can choose from.